dataactvalidator/interfaces/validatorStagingInterface.py

```diff
@@ -1,77 +1,52 @@
 from dataactcore.models.stagingInterface import StagingInterface
-from dataactcore.models.stagingModels import FieldNameMap
-from dataactcore.utils.responseException import ResponseException
-from dataactcore.utils.statusCode import StatusCode
-from dataactvalidator.interfaces.validatorJobTrackerInterface import ValidatorJobTrackerInterface
-from sqlalchemy import MetaData, Table
-from sqlalchemy.exc import NoSuchTableError
+from dataactcore.models.stagingModels import Appropriation, ObjectClassProgramActivity, AwardFinancial, AwardFinancialAssistance
 
 
 class ValidatorStagingInterface(StagingInterface):
     """ Manages all interaction with the staging database """
 
-    def dropTable(self,table):
+    def insertSubmissionRecordByFileType(self, record, fileType):
+        """Insert a submitted file row into staging.
+
+        Args:
+            record: record to insert (Dict)
+            submissionId: submissionId associated with this record (int)
+            fileType: originating file type of the record (string)
         """
+        if fileType == "award":
+            rec = AwardFinancialAssistance(**record)
+        elif fileType == "award_financial":
+            rec = AwardFinancial(**record)
+        elif fileType == "appropriations":
+            rec = Appropriation(**record)
+        elif fileType == "program_activity":
+            rec = ObjectClassProgramActivity(**record)
+        self.session.add(rec)
+        self.session.commit()
```

Comment:
@jworcestBAH You are right about the staging interface and the session handling, and I should stop doing this stuff late at night. I had tested my commit changes in the threaded validation manager, which wasn't being hit by the unit tests.

Just pushed a change to move the commit() out of the interface and up to the validation manager. I currently have it set to commit once at the end of the file.

To your point about chunking, I think we have fundamentally differing philosophies. I'm inclined not complicate the code base by putting in our own chunking logic until it becomes apparent that our AWS and database resources need us to. If we need to code it ourselves, I'd prefer to address when we hit DB-35.

I hope this doesn't come across as me being lazy. Your original point about burying the commit in the interface update function was spot on, and I'm glad you raised it. My goal is to keep the code base as simple as possible.